docs/modules/ROOT/pages/how-to/day2ops/disable-access.adoc

```diff
@@ -0,0 +1,41 @@
+= Disable access to {product}
+
+This guide describes how to remove a user's access to all {zone}s
+
+NOTE: This page will be updated once we've implemented support for selective access to single {zone}s.
+
+== Prerequisites
+
+* Installed {idp} (see xref:appuio-cloud:root:how-to/keycloak-setup.adoc[Install Keycloak])
```

Comment:
I'd remove this line as this isn't a concern when doing day 2 operations, this has already been done in day 0.